pkg/resources/marketplace/configmap_catalog_source_reconciler.go

```diff
@@ -0,0 +1,132 @@
+package marketplace
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+type ConfigMapCatalogSourceReconciler struct {
+	ManifestsProductDirectory string
+	Context                   context.Context
+	Client                    k8sclient.Client
+	Namespace                 string
+	CSName                    string
+}
+
```

Comment:
```suggestion

var _ CatalogSourceReconciler = &ConfigMapCatalogSourceReconciler{}

```

Blank statement to assert that `ConfgMapCatalogSourceReconciler` implements `CatalogSourceReconciler`

Reply:
:+1: I'll add it

Reply:
Done